### DISCUSSION TITLE
How to configure the components for building an agentic flow?

### DISCUSSION THREAD

**Discussion by FatemaD1577** [infiniflow/ragflow#4135] [Q&A]
I have been trying to build a flow that that will gather information in an iterative manner. Every time a peice of information is retrieved it should be analysed to identify missing details if any and the required information should be retrieved. Once all information is present than the answer generation should be done. I am not able to understand how to configure the inputs for different components. It will be helpful if more detailed document is available with guidance to create agentic flow.

**Comment by KevinHuSh** [comment]
It's more like the `SEO Blog` agent template which can be created from clicking `Create agent`.
I think you could get clue from this.
